conseil-common/src/main/resources/reference.conf

```diff
@@ -24,6 +24,8 @@ platforms: [
   {
     name: "tezos"
 
+    baker-rolls-size: 8000
```

Comment:
I would put it in the `conseil-lorre/src/main/resources/application.conf` - I think it's more suitable place for it as we have some more constants defined there.

Reply:
The only place in the code, where it's used are `MainOutputs`, and `TezosTypes` which are both located in `common`. All other usages are for testing.

We decided to leave the location as is.